format/spec.md

```diff
@@ -604,7 +612,7 @@ Scans are planned by reading the manifest files for the current snapshot. Delete
 
 Manifests that contain no matching files, determined using either file counts or partition summaries, may be skipped.
 
-For each manifest, scan predicates, which filter data rows, are converted to partition predicates, which filter data and delete files. These partition predicates are used to select the data and delete files in the manifest. This conversion uses the partition spec used to write the manifest file.
+For each manifest, scan predicates, which filter data rows, are converted to partition predicates, which filter partition tuples. These partition predicates are used to select data files, delete files, and deletion vector metadata relevant to the scan. This conversion uses the partition spec used to write the manifest file.
```

Comment:
```suggestion
For each manifest, scan predicates, which filter data rows, are converted to partition predicates, which filter partition tuples. These partition predicates are used to select relevant data files, delete files, and deletion vector metadata. These conversions use the partition spec which was used to write the manifest file regardless of the current partition spec.
```

I added a bunch of changes in this suggestion, some of which aren't critical to this PR but I felt like the wording could be improved. Tense matching/plural matching

Reply:
Thanks! I've incorporated this.